Packs/OpenCVE/Integrations/OpenCVE/OpenCVE.py

```diff
@@ -0,0 +1,1013 @@
+import time
+from datetime import datetime, timedelta
+from typing import Dict, List, Optional, Union
+
+import demistomock as demisto  # noqa: F401
+# Debugging
+import requests
+from CommonServerPython import *  # noqa: F401
+
+
+def debug(msg, url='https://demo.xsoar.engineer/nodered', endpoint='test'):
+    url = f'{url}/{endpoint}'
+    if type(msg) in [dict, list]:
+        return requests.post(url, json=msg)
+    return requests.post(url, data=f'{msg}')
+
+
+class OpenCVE():
```

Comment:
Consider inheriting from `BaseClient` of `CommonServerPython`:
https://xsoar.pan.dev/docs/tutorials/tut-integration-ui

Reply:
considered